src/seedu/addressbook/data/person/Address.java

```diff
@@ -6,11 +6,12 @@
  * Represents a Person's address in the address book.
  * Guarantees: immutable; is valid as declared in {@link #isValidAddress(String)}
  */
-public class Address {
+public class Address implements Printable{
 
     public static final String EXAMPLE = "123, some street";
     public static final String MESSAGE_ADDRESS_CONSTRAINTS = "Person addresses can be in any format";
     public static final String ADDRESS_VALIDATION_REGEX = ".+";
+    public static final String COMMON_WORD = "Address: ";
 
```

Comment:
Perhaps you can give a more meaningful / insightful name that give readers some idea of the string.